testbed/datasenders/tcp.go

```diff
@@ -0,0 +1,131 @@
+// Copyright  OpenTelemetry Authors
```

Comment:
Can we rename this file to tcpudp.go to better reflect the intent?

Reply:
fixed